src/views/assets/Platform/PlatformList.vue

```diff
@@ -112,8 +112,8 @@ export default {
               canUpdate: ({ row }) => !row.internal && vm.$hasPerm('assets.change_platform'),
               canDelete: ({ row }) => !row.internal && vm.$hasPerm('assets.delete_platform'),
               onUpdate({ row, col }) {
-                vm.$route.query.type = row.type.value
-                vm.$route.query.category = row.category.value
+                vm.$route.query._type = row.type.value
+                vm.$route.query._category = row.category.value
                 vm.$refs.genericListTable.onUpdate({ row, col })
               }
             }
```

Comment:
It seems that there is no clear difference between the two versions of the code snippet provided here. There might be minor syntax or formatting inconsistencies but nothing crucial that would require significant modifications.

As such, I don't see any irregularities, potential issues, nor suggestions for optimization at this stage without additional context or details about what exactly you're looking to achieve with these changes. To provide more specific feedback on whether it needs tweaking, please share details like:

- The exact purpose (or problem) it's intended to solve.
- The features that specifically need improvement compared to the old version.
  
Remember I've used `@` symbol in code snippets due lack of real world examples from which to derive conclusions.